README.md

```diff
@@ -194,20 +199,19 @@ contract DonationsV2 is DonationsV1 {
 }
 ```
 
-What we need to do next is link our application to the zOS Kernel standard library release containing that mintable ERC721 implementation, and set it to our upgradeable contract. To do so, we create a new version of our application in the `AppManager`, register a new `AppDirectory` containing the new version of our contract implementation, and then set the standard library version of ERC721 to our upgradeable contract. Let's see how:
+5. What we need to do next is link our application to the zOS Kernel standard library release containing that mintable ERC721 implementation, and set it to our upgradeable contract. To do so, we create a new version of our application in the `AppManager`, register a new `AppDirectory` containing the new version of our contract implementation, and then set the standard library version of ERC721 to our upgradeable contract. Let's see how:
 
 ```js
-  // Create a new version of the app, liked to the zeppelin_os standard library.
-  // Register a new implementation for "Basil" and upgrade it's proxy to use the new implementation.
-  log("\n<< Deploying version 2 >>")
+  // Address of the zOS Kernel standard library.
+  const stdlib = "0xA739d10Cc20211B973dEE09DB8F0D75736E2D817";
```

Comment:
where does this come from?

Reply:
From `examples/complex/index.js`. The var was used two lines below, without being defined or explained.